### PR TITLE
feat: add view scheduler job component

### PR DESCRIPTION
**Pull request by BLasan**
**Description**
Component to view selected Job.

**Related issues and discussion**
#673 

**Screenshots, if any**
Screenshot from 2020-05-12 15-19-51

**Checklist**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   * @param {ActivatedRoute} route Activated Route.
+   */
+  constructor(private route: ActivatedRoute) {
+    this.route.data.subscribe((data: { selectedJob: any }) => {
+      this.jobData = data.selectedJob;
+    });
+  }
+
+  ngOnInit() {
+
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts]:
```suggestion

```

File: src/app/system/system.service.ts [hunk 2]
```diff
@@ -136,6 +136,14 @@ export class SystemService {
     return this.http.get('/scheduler');
   }
 
+  /**
+   * @param jobId  Selected Job Id
+   * @returns {Observable<any>}
+   */
+  getSelctedJob(jobId: string) {
```
Comment (karantakalkar) [inline src/app/system/system.service.ts]:
```suggestion
  getSelectedJob(jobId: string) {
```

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+   */
+  constructor(private systemService: SystemService) {}
+
+  /**
+   * Returns the selected scheduler job data.
+   * @returns {Observable<any>}
+   */
+  resolve(route: ActivatedRouteSnapshot): Observable<any> {
+    const jobId = route.paramMap.get('id');
+    return this.systemService.getSelctedJob(jobId);
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.resolver.ts]:
```suggestion
    return this.systemService.getSelectedJob(jobId);
```

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { SystemService } from 'app/system/system.service';
+
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts]: add comment view scheduler job component

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { SystemService } from 'app/system/system.service';
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts]: add comment custom services

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts]: add comment angular imports

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts (hunk 4 above)
Comment (BLasan) [reply src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.ts]: Please check again. I commented for the first two imports and removed the third one

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.html:1]: make the structure of this html consistent with view tax component.html, take care of indentations.

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+a{
+    color: blue;
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss:2]:
```suggestion
 color: blue;
```

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
...
+}
```
Comment (karantakalkar) [inline src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss:3]: new line at eof

File: src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss (hunk 8 above)
Comment (BLasan) [reply src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss:2]: Should I align straight vertically down to the 'a' tag?

Comment (karantakalkar) [reply src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss:2]: no just reduce indent a bit

Comment (BLasan) [reply src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss:2]: Adjusted the alignment

**Comment by karantakalkar** [comment]
squash commits

**Comment by BLasan** [comment]
> squash commits

Sorry, mistaken

**Pull request merged**